shared/versions.yml

```diff
@@ -70,18 +70,34 @@ couchdb:
 dataloader:
   upstream:
     repository: gpii/universal
-    tag: 20190522142238-4a52f56
+    tag: 20190610144430-9bd021f
```

Comment:
I'm slightly worried about using 2 different Universal images at the same time (one being used for dataloader/flushtokens, the other one for preferences/flowmanager). What do others think? @mrtyler @amatas @natarajaya 

I'm aware that updating all the images would mean bringing other changes in, but perhaps making this PR dependant on the universal update PR, and then using the same version everywhere would be better.

Reply:
I _think_ there is an historical artefact in this case.  Originally, the version of universal used for `gpii-flushtokens` was my docker image (`herrclown/vagrant-universal`) and that's because that image included the access-token deletion scripts (`dockerDeleteExpiredAccessTokens.sh` and `deleteExpiredAccessTokens.js`).

Later, when those two scripts were merged into universal's master branch, I switched the tag to use that version of universal -- I don't think that gpii-infra was using that version of universal "yet".  Maybe it does now.  I haven't paid that much attention to versions since I wanted to spend my efforts on debugging the issue at hand, that being: getting rid of the sidecar.

All that's needed, in terms of a universal image, is one with the latest flowmanager, preferences, dataloader, and flushtokens (anything else?) modules.

Reply:
Ok then, so what about adding a note that this depends on the newly created https://github.com/gpii-ops/gpii-infra/pull/438 - I believe the image there should have your changes - and leaving the image updates out of this PR?

Reply:
Okay, where does the note go?  I could put it in the description of this pull request, or in the JIRA (GPII-3888), or both.

Looking at the [version.yml](https://github.com/the-t-in-rtf/gpii-infra/blob/GPII-4006/shared/versions.yml#L72) in PR #438, the `repository` and `tags` are:

```
repository: gpii/universal
tag: 20190627203047-1262ac8
```
Is that correct?

Reply:
Yes description of the PR is the place 👍 

That is correct, although I expect there'll be even newer tag given there are still some performance issues.